src/LinkDotNet.NCronJob/RetryPolicies/FixedIntervalRetryPolicyCreator.cs

```diff
@@ -0,0 +1,34 @@
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Logging;
+using Polly;
+
+namespace LinkDotNet.NCronJob;
+
+/// <summary>
+/// A policy creator that configures a fixed interval retry policy.
+/// </summary>
+public partial class FixedIntervalRetryPolicyCreator : IPolicyCreator, IInitializablePolicyCreator
```

Comment:
Does that have to be `public`?

Reply:
Same as above. No, only if we want to give users access to it.